### PR TITLE
unnest social link markup

### PR DESCRIPTION
**Pull request by mndvns**
@jake this unnests the social links so they aren't wrapped in an extra div. this makes styling much easier. i tested the change in the list-ui app, and it doesn't affect styles in the `SocialList` component.

### CODE REVIEW

File: blocks/el/social-link/index.jade [hunk 1]
```diff
@@ -12,17 +12,16 @@ const FB_APP_ID = browser.env.FB_APP_ID
...
+  socialAttrs = poll
+    ? pollSocialAttrs
+    : quiz
+      ? quizSocialAttrs
+      : listSocialAttrs
 
-
-export var mergeProps = true
+if item
+  SocialLinkTemplate()&props(merge(socialAttrs($get, t, item, fbAppId), props))
```
Comment (jakerichan) [inline blocks/el/social-link/index.jade]: missing quiz result

File: blocks/el/social-link/index.jade [hunk 2]
```diff
@@ -12,17 +12,16 @@ const FB_APP_ID = browser.env.FB_APP_ID
 const BOOMBOX_FB_APP_ID = browser.env.BOOMBOX_FB_APP_ID
```
Comment (jakerichan) [inline blocks/el/social-link/index.jade]: BTW - don't forget to add this to the list apps

File: blocks/el/social-link/index.jade (hunk 1 above)
Comment (jakerichan) [inline blocks/el/social-link/index.jade]: maybe we convert this to a single object argument

Comment (mndvns) [inline blocks/el/social-link/index.jade]: ah damnit. _sigh_
i'm going to have to nest the css until we get this resolved. damnit

**Pull request closed without merging**